invenio_administration/assets/semantic-ui/js/invenio_administration/actions/Cancel.js

```diff
@@ -0,0 +1,29 @@
+import React, { Component } from "react";
+import PropTypes from "prop-types";
+import { i18next } from "@translations/invenio_administration/i18next";
+import { Button } from "semantic-ui-react";
+import { omit } from "lodash";
+
+export class Cancel extends Component {
+  render() {
+    const { negative } = this.props;
+    const props = omit(this.props, ["negative"]);
```

Comment:
why do we omit the prop? it is used below

Reply:
Not sure what I was thinking here. Thanks